build/docker/docker-compose-test.yml

```diff
@@ -0,0 +1,112 @@
+version: "3.7"
+
+services:
+  test-db:
+    image: $DOCKER_REGISTRY/data:latest
+    container_name: test-db
+    hostname: test-db
+    environment:
+      PROFILE: $PROFILE
+      POSTGRES_DB: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      PGPASSWORD: postgres
+    ports:
+      - "5432:5432"
+    command: ["postgres", "-c", "config_file=/etc/postgresql/postgresql.conf"]
+    networks:
+      default:
+        aliases:
+          - test-db.$PROJECT_NAME_SHORT.local
+
+  data:
+    image: $DOCKER_REGISTRY/data:latest
+    container_name: data
+    hostname: data
+    environment:
+      PROFILE: $PROFILE
+      POSTGRES_DB: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      PGPASSWORD: postgres
+      PGHOST: test-db.$PROJECT_NAME_SHORT.local
+      PGPORT: 5432
+      PGUSERNAME: postgres
+      PGDATABASENAME: postgres
+      SQL_DIR: /sql
+    command: ["sh", "/script/load_dos_db.sh"]
```

Comment:
Please, convert that line to `command: ["sql"]` which will execute any SQL script in the `/sql` directory.